src/main/java/Main.java

```diff
@@ -0,0 +1,15 @@
+public class Main {
+    public static void main(String[] args) {
```

Comment:
remove redundant empty lines from this file

Reply:
Hello! 
Thank you for answer!
I added new branch and the next tasks will be there :-) 
Please check out last commit. I made changes.